Scripts/Main.py

```diff
@@ -0,0 +1,52 @@
+import Sorcery_Functions
+
+# Testing our SorceryFunctions module
+
+# The below if statement is added in case we want to input
+# arguments in the command line for testing the Sorcery function.
+# So it's mainly added for convenience, since an alternative
+# would be to manually edit this script. When command line
+# arguments are used, the Main function expects the first
+# argument to be a boolean, although if this is not the case
+# it will type cast the argument to type "bool"
+# if (__name__ == "__main__"):
+#     import sys
+#     includeForbiddenSpells = bool(sys.argv[1])
+#     createSpells = Sorcery_Functions.createSpells
+#     printSpellBook = Sorcery_Functions.printSpellBook
```

Comment:
Lines 13 to 16 are over indented.

I am also running an error on line 14 (Line 15 for me since I added a blank line locally)

Traceback (most recent call last):
  File "Scripts/Main.py", line 15, in <module>
    includeForbiddenSpells = bool(sys.argv[1])
IndexError: list index out of range